### PR TITLE
Update Java 22 specific test with GA version

### PR DESCRIPTION
**Pull request by gjwatts**
Updating the Java 22 build to use the GA version of Java 22.

### CODE REVIEW

**Comment by gjwatts** [comment]
The Java 22 build failure is expected since there is no Java 22 Temurin version available (yet).

**Review by KyleAure — APPROVED** [review]

**Comment by gjwatts** [comment]
Waiting to merge until a Temurin Java 22 build is available.

**Comment by gjwatts** [comment]
#build

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
   build:
     strategy:
       matrix:
-        java-version: [ '17', '18', '19', '20', '21', '22-ea' ]
```
Comment (KyleAure) [inline .github/workflows/CI.yml:16]: Probably want to wait until the actions/setup-java is updated with Java 22 GA support.

Comment (gjwatts) [reply .github/workflows/CI.yml:16]: Agreed

**Review by KyleAure — APPROVED** [review]

**Comment by gjwatts** [comment]
Switched to using oracle versions and that worked so we no longer need to wait for a Java 22 Temurin version.

**Pull request merged**